### PR TITLE
Upgrades to CDH5

### PR DESCRIPTION
**Pull request by stephanh**
Also enables testing for the tools subproject again.

### CODE REVIEW

File: tools/src/test/scala/au/com/cba/omnia/ebenezer/introspect/ParquetIntrospectToolsSpec.scala [hunk 1]
```diff
@@ -60,7 +60,7 @@ Introspect on all types
   Read nested                               ${check("nested", fromNestedish)}
   Read lists                                ${check("list", fromListish)}
   Read maps                                 ${check("map", fromMapish)}
-  Read emums                                ${check("enum", fromEnumish)}
+  Read emums                                $pending //{check("enum", fromEnumish)}
```
Comment (vineethvarghese) [inline tools/src/test/scala/au/com/cba/omnia/ebenezer/introspect/ParquetIntrospectToolsSpec.scala]: Didn't know comments work in interpolated strings

File: tools/src/test/scala/au/com/cba/omnia/ebenezer/introspect/ParquetIntrospectToolsSpec.scala [hunk 2]
```diff
@@ -125,8 +125,9 @@ Introspect on all types
     }):_*)))
   ))
 
+  // Enum support is broken somewhere See https://github.com/CommBank/ebenezer/issues/49
   def fromEnumish: Enumish => Record = d => Record(List(
-    Field("value", EnumValue(d.value.name.toUpperCase))
+     Field("value", EnumValue(d.value.name.toUpperCase))
```
Comment (vineethvarghese) [inline tools/src/test/scala/au/com/cba/omnia/ebenezer/introspect/ParquetIntrospectToolsSpec.scala]: Extra indentation

File: tools/src/test/scala/au/com/cba/omnia/ebenezer/introspect/ParquetIntrospectToolsSpec.scala (hunk 1 above)
Comment (stephanh) [inline tools/src/test/scala/au/com/cba/omnia/ebenezer/introspect/ParquetIntrospectToolsSpec.scala]: :smile: I don't think it does. Just prints the string. I'll fix it up.

File: project/build.scala [hunk 3]
```diff
@@ -115,11 +112,11 @@ object build extends Build {
...
         ++ Seq(
           libraryDependencies ++=
-            depend.hadoop() ++ depend.scalding() ++ depend.scalaz() ++ depend.testing() ++ Seq(
-              "com.twitter" % "parquet-tools" % "1.5.0"
+            depend.hadoop() ++ depend.scalaz() ++ depend.testing() ++ depend.parquet() ++ Seq(
+              "com.twitter" % "parquet-tools" % depend.versions.parquet exclude("org.apache.hadoop", "hadoop-client")
             )
         )
-  ).dependsOn(core)
+  ).dependsOn(core % "test->test")
```
Comment (vineethvarghese) [inline project/build.scala]: What does "core % "test->test"" mean?

Comment (stephanh) [inline project/build.scala]: It says that the test task of this particular subproject depends on the task task of the core project.

File: tools/src/test/scala/au/com/cba/omnia/ebenezer/introspect/ParquetIntrospectToolsSpec.scala (hunk 2 above)
Comment (rowandavies) [inline tools/src/test/scala/au/com/cba/omnia/ebenezer/introspect/ParquetIntrospectToolsSpec.scala]: Indented by 3 - it should be 2 to match the subsequent code.

**Comment by vineethvarghese** [comment]
A few minor things but otherwise :+1: 

The build is failing though.

File: .travis.yml [hunk 4]
```diff
@@ -12,7 +12,7 @@ install:
 - ci/sbt-setup.sh
 - ci/sbt-setup-version.sh
 script:
-- sbt -Dsbt.global.base=$TRAVIS_BUILD_DIR/ci '; test; package; project example; test; assembly; project compat; test; project tools; assembly'
+- sbt -Dsbt.global.base=$TRAVIS_BUILD_DIR/ci '; test; package; project example; test; assembly; project compat; test; project tools; test; assembly'
```
Comment (rowandavies) [inline .travis.yml]: So the tests for the tools project weren't being run previously?  If so, nice catch.

**Comment by rowandavies** [comment]
Looks fine to me, aside from the build issue.

**Comment by stephanh** [comment]
@vineethvarghese and @rowandavies I rewrote some of the tests. Do you want to have another look?

**Comment by vineethvarghese** [comment]
Looking now.

**Comment by vineethvarghese** [comment]
:+1:  The test sure looks better.

**Comment by stephanh** [comment]
Thank you

**Pull request merged**